engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -1872,12 +1895,14 @@ class IrToTruffle(
     ) {
       private val argFactory = new DefinitionArgumentProcessor(scopeName, scope)
       private lazy val slots = computeSlots()
-      private lazy val bodyN = computeBodyNode()
+      lazy val argsExpr      = computeArgsAndExpression()
```

Comment:
The name is an internal detail.

Reply:
Yes,  impact of local variable naming is limited. Whatever the name is, it doesn't propagate to the rest of the system. 

> Internal details are just as important to be understandable,

No, internal details are not as important from an architecture perspective. From an architecture perspective it is important to care more about things visible from outside and influencing other part of system. Public names, classes being public instead of (package) private, set of dependencies, etc. is what matters, not perfect local variable naming is a niche.

**Last note:** Given I had spent three days CI fighting I decided to focus on getting my change in, rather than cleaning this little detail and then waiting for all the CI actions to start from scratch and randomly fail again.